results/test-extrinsic-100-summaries.json

```diff
@@ -1 +1 @@
6406":false,"17996567":false,"18239960":false,"19541955":false,"20191016":false,"25285256":false,"26398483":false,"28071499":false,"28772094":false,"29252950":false,"30457745":false,"30657223":false,"30760966":false,"31113635":false,"32033856":false,"32132840":false,"32487884":false,"32557344":false,"32651759":false,"32798693":false,"33044932":false,"33106194":false,"33317600":false,"33648595":false,"34193568":false,"34427234":false,"34474416":false,"34577604":false,"34614991":false,"34660292":false,"34694547":false,"34760893":false,"34787804":false,"34898778":false,"35038754":false,"35159263":false,"35237411":false,"35305028":false,"35308608":false,"35360415":false,"35369658":false,"35411078":false,"35698210":false,"35723757":false,"35811486":false,"35845462":false,"35972298":false,"36000903":false,"36061830":false,"36066252":false,"36162485":false,"36295288":false,"36301742":false,"36327891":false,"36341722":false,"36540012":false,"36555657":false,"36635880":false,"36730443":false,"36772094":false,"36874565":false,"36912060":false,"36932519":false,"36961338":false,"37310327":false,"37327555":false,"37391662":false,"37603756":false,"37604251":false,"37870098":false,"37961975":false,"38003317":false,"38078754":false,"38113695":false,"38219213":false,"38299022":false,"38399324":false,"38414086":false,"38470818":false,"38779380":false,"38983532":false,"39431070":false,"39535308":false,"39556625":false,"39582812":false,"39593375":false,"39753996":false,"39784504":false,"39807248":false,"39847765":false,"39895178":false,"39959265":false,"40243246":false,"40285040":false,"40306866":false,"40426136":false,"40498102":false,"40551256":false,"40587903":false},"gold_n_extrinsic":{"14702705":0,"14766406":0,"17996567":0,"18239960":0,"19541955":0,"20191016":0,"25285256":0,"26398483":0,"28071499":0,"28772094":0,"29252950":0,"30457745":0,"30657223":0,"30760966":0,"31113635":0,"32033856":0,"32132840":0,"32487884":0,"32557344":0,"32651759":0,"32798693":0,"33044932":0,"33106194":0,"33317600":0,"33648595":0,"34193568":0,"34427234":0,"34474416":0,"34577604":0,"34614991":0,"34660292":0,"34694547":0,"34760893":0,"34787804":0,"34898778":0,"35038754":0,"35159263":0,"35237411":0,"35305028":0,"35308608":0,"35360415":0,"35369658":0,"35411078":0,"35698210":0,"35723757":0,"35811486":0,"35845462":0,"35972298":0,"36000903":0,"36061830":0,"36066252":0,"36162485":0,"36295288":0,"36301742":0,"36327891":0,"36341722":0,"36540012":0,"36555657":0,"36635880":0,"36730443":0,"36772094":0,"36874565":0,"36912060":0,"36932519":0,"36961338":0,"37310327":0,"37327555":0,"37391662":0,"37603756":0,"37604251":0,"37870098":0,"37961975":0,"38003317":0,"38078754":0,"38113695":0,"38219213":0,"38299022":0,"38399324":0,"38414086":0,"38470818":0,"38779380":0,"38983532":0,"39431070":0,"39535308":0,"39556625":0,"39582812":0,"39593375":0,"39753996":0,"39784504":0,"39807248":0,"39847765":0,"39895178":0,"39959265":0,"40243246":0,"40285040":0,"40306866":0,"40426136":0,"40498102":0,"40551256":0,"40587903":0}}
```

Comment:
I feel like JSONL would lead to better diffs! Hard to parse the change, but /shrug